packages/discord.js/src/util/BitField.js

```diff
@@ -164,7 +164,11 @@ class BitField {
     const { DefaultBit } = this;
     if (typeof DefaultBit === typeof bit && bit >= DefaultBit) return bit;
     if (bit instanceof BitField) return bit.bitfield;
-    if (Array.isArray(bit)) return bit.map(p => this.resolve(p)).reduce((prev, p) => prev | p, DefaultBit);
+    if (Array.isArray(bit)) {
+      return bit
+        .map(permission => this.resolve(permission))
+        .reduce((prev, permission) => prev | permission, DefaultBit);
```

Comment:
Something like this is preferrable, as this is the generic BitField class, which doesn't necessarily mean this will be an array of permissions

`    if (Array.isArray(bit)) return bit.map(bit_ => this.resolve(bit_)).reduce((prev, bit_) => prev | bit_, DefaultBit);`